ClemBot.Bot/bot/cogs/custom_tag_prefix_cog.py

```diff
@@ -0,0 +1,82 @@
+import logging
```

Comment:
This entire file should be in the tags_cog.py file, as it is a tags command. they will be child commands on the !tag group